Get 8 build errors on Linux

Get 8 errors when building latest corefx on ubuntu with dnx-coreclr-linux-x64.1.0.0-rc1-15838.
```
CSC : error CS7027: Error signing output with public key from file 
'/data/git/corefx/packages/Microsoft.DotNet.BuildTools.1.0.25-prerelease-00104/lib/ECMA.snk' 
-- mscoree.dll [/data/git/corefx/src/System.IO.Compression.ZipFile/ref/System.IO.Compression.ZipFile.csproj]

CSC : error CS7033: Delay signing was specified and requires a public key, but no public key was specified 
[/data/git/corefx/src/System.IO.Compression.ZipFile/ref/System.IO.Compression.ZipFile.csproj]

CSC : error CS7027: Error signing output with public key from file 
'/data/git/corefx/packages/Microsoft.DotNet.BuildTools.1.0.25-prerelease-00104/lib/ECMA.snk' 
-- mscoree.dll [/data/git/corefx/src/System.IO.Compression/ref/System.IO.Compression.csproj]

CSC : error CS7033: Delay signing was specified and requires a public key, but no public key was specified 
[/data/git/corefx/src/System.IO.Compression/ref/System.IO.Compression.csproj]

CSC : error CS7027: Error signing output with public key from file 
'/data/git/corefx/packages/Microsoft.DotNet.BuildTools.1.0.25-prerelease-00104/lib/ECMA.snk' 
-- mscoree.dll [/data/git/corefx/src/System.Reflection.Context/ref/System.Reflection.Context.csproj]

CSC : error CS7033: Delay signing was specified and requires a public key, but no public key was specified 
[/data/git/corefx/src/System.Reflection.Context/ref/System.Reflection.Context.csproj]

CSC : error CS7027: Error signing output with public key from file 
'/data/git/corefx/packages/Microsoft.DotNet.BuildTools.1.0.25-prerelease-00104/lib/ECMA.snk' 
-- mscoree.dll [/data/git/corefx/src/System.Runtime.WindowsRuntime.UI.Xaml/ref/System.Runtime.WindowsRuntime.UI.Xaml.csproj]

CSC : error CS7033: Delay signing was specified and requires a public key, but no public key was specified 
[/data/git/corefx/src/System.Runtime.WindowsRuntime.UI.Xaml/ref/System.Runtime.WindowsRuntime.UI.Xaml.csproj]
```

Copying the snippet from @jasonwilliams200OK's log:
```
 /root/corefx/packages/Microsoft.Build.Mono.Debug.14.1.0.0-prerelease/lib/tools/../tools/csc2.exe -noconfig -nowarn:1701,1702 -langversion:5 -nostdlib+ -warn:4 -define:DEBUG;TRACE;DEBUGRESOURCES;SIGNED -reference:/root/corefx/packages/System.IO.Compression/4.0.0/ref/dotnet/System.IO.Compression.dll -reference:/root/corefx/packages/System.IO/4.0.0/ref/dotnet/System.IO.dll -reference:/root/corefx/packages/System.Runtime/4.0.0/ref/dotnet/System.Runtime.dll -reference:/root/corefx/packages/System.Text.Encoding/4.0.0/ref/dotnet/System.Text.Encoding.dll -debug- -delaysign+ -keyfile:/root/corefx/packages/Microsoft.DotNet.BuildTools.1.0.25-prerelease-00104/lib/ECMA.snk -optimize- -out:/root/corefx/bin/obj/ref/System.IO.Compression.ZipFile/4.0.1.0/System.IO.Compression.ZipFile.dll -target:library -warnaserror+ System.IO.Compression.ZipFile.cs /root/corefx/bin/obj/ref/System.IO.Compression.ZipFile/4.0.1.0/_AssemblyInfo.cs
CSC : error CS7027: Error signing output with public key from file '/root/corefx/packages/Microsoft.DotNet.BuildTools.1.0.25-prerelease-00104/lib/ECMA.snk' -- mscoree.dll [/root/corefx/src/System.IO.Compression.ZipFile/ref/System.IO.Compression.ZipFile.csproj]
CSC : error CS7033: Delay signing was specified and requires a public key, but no public key was specified [/root/corefx/src/System.IO.Compression.ZipFile/ref/System.IO.Compression.ZipFile.csproj]
```

I assume the file `/root/corefx/packages/Microsoft.DotNet.BuildTools.1.0.25-prerelease-00104/lib/ECMA.snk` exists? 

I'm not at all sure what is happening here. @akoeplinger @ellismg any ideas? My only guess is there is something going on with mono csc here.

--- a/src/System.IO.Compression.ZipFile/ref/System.IO.Compression.ZipFile.csproj
+++ b/src/System.IO.Compression.ZipFile/ref/System.IO.Compression.ZipFile.csproj
@@ -1,8 +1,8 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
-    <UseECMAKey>true</UseECMAKey>
+    <UseECMAKey Condition="'$(UseECMAKey)' == ''">true</UseECMAKey>
     <AssemblyVersion>4.0.1.0</AssemblyVersion>
     <OutputType>Library</OutputType>
   </PropertyGroup>

--- a/src/System.IO.Compression/ref/System.IO.Compression.csproj
+++ b/src/System.IO.Compression/ref/System.IO.Compression.csproj
@@ -2,7 +2,7 @@
 <Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
-    <UseECMAKey>true</UseECMAKey>
+    <UseECMAKey Condition="'$(UseECMAKey)' == ''">true</UseECMAKey>
     <AssemblyVersion>4.1.0.0</AssemblyVersion>
     <OutputType>Library</OutputType>
   </PropertyGroup>

--- a/src/System.Reflection.Context/ref/System.Reflection.Context.csproj
+++ b/src/System.Reflection.Context/ref/System.Reflection.Context.csproj
@@ -1,8 +1,8 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
-    <UseECMAKey>true</UseECMAKey>
+    <UseECMAKey Condition="'$(UseECMAKey)' == ''">true</UseECMAKey>
     <AssemblyVersion>4.0.1.0</AssemblyVersion>
     <OutputType>Library</OutputType>
   </PropertyGroup>

--- a/src/System.Runtime.WindowsRuntime.UI.Xaml/ref/System.Runtime.WindowsRuntime.UI.Xaml.csproj
+++ b/src/System.Runtime.WindowsRuntime.UI.Xaml/ref/System.Runtime.WindowsRuntime.UI.Xaml.csproj
@@ -1,8 +1,8 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
-    <UseECMAKey>true</UseECMAKey>
+    <UseECMAKey Condition="'$(UseECMAKey)' == ''">true</UseECMAKey>
     <AssemblyVersion>4.0.1.0</AssemblyVersion>
     <OutputType>Library</OutputType>
   </PropertyGroup>

--- a/src/System.Runtime.WindowsRuntime/ref/System.Runtime.WindowsRuntime.csproj
+++ b/src/System.Runtime.WindowsRuntime/ref/System.Runtime.WindowsRuntime.csproj
@@ -1,8 +1,8 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
-    <UseECMAKey>true</UseECMAKey>
+    <UseECMAKey Condition="'$(UseECMAKey)' == ''">true</UseECMAKey>
     <!-- 
          NOTE: Suppress false positive warning for the special case where we're building System.Runtime.WindowsRuntime itself
                at a version other than 4.0.0.0, which is referenced indirectly via the mscorlib.dll design-time facade, which

